dask/dataframe/io/parquet.py

```diff
@@ -0,0 +1,170 @@
+import os
+import struct
+
+from toolz import first
+
+import dask
+from dask import delayed, compute
+from dask.bytes.core import open_files, OpenFileCreator
+import dask.dataframe as dd
+from dask import delayed
+
+try:
+    import fastparquet
+    from fastparquet import parquet_thrift
+    default_encoding = parquet_thrift.Encoding.PLAIN
+except:
+    fastparquet = False
+    default_encoding = None
```

Comment:
I came up with this, but it seems a little ugly. Is there a more standard way to avoid import errors, where we need a couple of references in the function signatures below?